vcluster/_fragments/sleepmode-istio-example.mdx

```diff
@@ -0,0 +1,116 @@
+import Flow, { Step } from '@site/src/components/Flow';
+import CodeBlock from '@theme/CodeBlock'
+import SleepmodeIstioConfig from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/_code/sleepmode-istio-config.yaml'
+import SleepmodeIstioExample from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/experimental/_code/sleepmode-istio-examples.yaml'
+
+### Configure sleep mode for use with Istio
+
+<Flow id="sleepmode-ingress-example">
+  <Step>
+Create the `kind` cluster.
+
+```shell title="Create a kind cluster"
+kind create cluster --name istio-demo --config - <<EOF
```

Comment:
How much time would it take to switch this from kind to a real cluster ? Or have both side by side. 
If it takes a while, for sake of timing, we could merge this one and do that later on.

Reply:
It's simpler on a real cluster, or at least on OrbStack where I did most of my dev/testing.  I wrote the example in kind to make clean up easier.

Reply:
Scratch that, at least if we want the instructions to deal with ambient mode specifically, in the interest of time we should leave it with kind.

Reply:
Ok, can you work on that later this week. Feel free to open up new issue.